lib/sinon/util/is_function.js

```diff
@@ -0,0 +1,12 @@
+// Just a typeof check, but accepts oldIE native funcs (their typeof is "object")
+module.exports = function isFunction(func) {
```

Comment:
This could be simplified a bit.

````` javascript
module.exports = function(object) {
  return typeof object == 'function' || typeof object == 'object' && object !== null && 
    /^\s*function[^{]*{\s*\[native code\]\s*}\s*$/i.test(object.toString())
  ;
};
````